utils/default-helpers/src/main/scala/pl/touk/nussknacker/engine/util/functions/conversion.scala

```diff
@@ -138,6 +139,30 @@ trait ConversionUtils extends HideToString {
     case _                       => null
   }
 
+  @Documentation(description = "Convert String value to JSON")
+  def toJson(@ParamName("value") value: String): Any = {
+    toJsonEither(value).toTry.get
+  }
+
+  @Documentation(description = "Convert String value to JSON or null in case of failure")
+  def toJsonOrNull(@ParamName("value") value: String): Any = {
+    toJsonEither(value).getOrElse(null)
+  }
+
+  @Documentation(description = "Convert JSON to String")
+  def toJsonString(@ParamName("value") value: Any): String = {
+    jsonEncoder.encode(value).noSpaces
+  }
+
+  private def toJsonEither(value: String): Either[Throwable, Any] = {
+    io.circe.parser.parse(value) match {
+      case Right(json) => Right(JsonUtils.jsonToAny(json))
+      case Left(ex)    => Left(new IllegalArgumentException(s"Cannot convert [$value] to JSON", ex))
```

Comment:
Hmm, I'm not sure, but here we should throw Transient exception?

Reply:
AFAIK by default all exceptions except `ConnectException` are treaded as NonTransient exceptions which are not propagated to flink from interpreter and don't cause scenario restart.
The difference is that exceptions wrapped inside `NonTransientException` are not logged as unknown exceptions in `WithExceptionExtractor.extractOrThrow`.

We actually don't mark errors thrown from utils as NonTransientExceptions so IMO marking only this one doesn't make sense.